### DISCUSSION TITLE
Unity package

### DISCUSSION THREAD

**Discussion by CryptoJanne** [ravendb/ravendb#13558] [Ideas]
I would love for RavenDB to make their own unity package. 

The use cases are obvious, RavenDB is perfect for game development. Both Unity and RavenDB uses c#, so any interaction with the API are easy to understand for someone who develops using unity.

I have found some people online getting it to work, but I did not manage myself.  I dont think it would take much time to make, but what do i know.

Thoughts?

**Comment by garayx** [comment]
Hi 

What issues did you run into ? Did you have any exceptions ? Can you share the stack trace ?

**Reply by omerlewitz** [reply]
@CryptoJanne I will pass along your suggestion of creating a dedicated `unity package`.

In general, we support `.Net Core Standards` so the next workaround should work.

These are the steps to make it work - 
- Download our `RavenDB-Client` from `NuGet Gallery` 
- Change the file extension from `.nupkg` to `.zip`
- Create a new `plugin` folder inside the `Assets` folder and copy the  `Raven.Client.dll` file.
